examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/index.html

```diff
@@ -7,9 +7,9 @@
 		Checkbox
 	</title>
 	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```

Comment:
until final decision is made please don't remove media queries in existing items

Reply:
Ok restored.